src/SyntaxHighlighter/X86Assembly.cpp

```diff
@@ -11,6 +11,18 @@
 #include <QString>
 #include <QTextCharFormat>
 
+// TODO: What is the best way to do this? Move the Metadata to another file? Declare it as extern?
```

Comment:
Yes, a separate file is probably best. But also a lot of work, because it needs to go into its own cmake module.

I'm thinking it might be better to detect an assemble line by looking at the code. All assembly lines start with the address `0x...`.

Reply:
As discussed offline, this won't work properly because the first lines in the assembly file will be highlight as C++ lines. We decided to not go in this direction.